tiledb/sm/group/group.cc

```diff
@@ -323,7 +308,7 @@ void Group::delete_group(const URI& uri, bool recursive) {
         storage_manager_->delete_array(member_uri.to_string().c_str());
       } else if (member->type() == ObjectType::GROUP) {
         Group group_rec(member_uri, storage_manager_);
-        throw_if_not_ok(group_rec.open(QueryType::MODIFY_EXCLUSIVE));
+        group_rec.open(QueryType::MODIFY_EXCLUSIVE);
         group_rec.delete_group(member_uri, true);
```

Comment:
Turns out it gets closed by `delete_group`. I tried to use the RAII wrapper and got test failures because the group is already closed. Let's keep this code as-is for now.

Reply:
If `delete_group` fails, it won't be closed. That situation would be a resource leak.

Reply:
Let's issues like this in the moment, unless it requires extensive refactoring (I don't think that's quite the case here -- but in those cases please open a story and flag for scheduling discussion).

Reply:
I found a way, and opened SC-27339 for next.